bridge_adaptivity/module/models.py

```diff
@@ -52,11 +52,17 @@ def _load_cls_from_applicable_module(module_path, mod_name, class_startswith=Non
 }
 GRADING_POLICY_CHOICES = ((k, v.public_name) for k, v in GRADING_POLICY_NAME_TO_CLS.items())
 
+PROBLEM_ACTIVITY_TYPES = [
+    'problem',
+]
+
 
 @python_2_unicode_compatible
 class Sequence(models.Model):
     """Represents User's problem solving track."""
 
+    PROBLEM_ACTIVITY_TYPES = PROBLEM_ACTIVITY_TYPES
```

Comment:
Why line 64 was added?

Reply:
to be able to get problem activity types from sequence instance. I use it in mock engine